containers/daap-athena-load/src/var/task/create_raw_athena_table.py

```diff
@@ -13,18 +13,17 @@ def create_raw_athena_table(
     Creates an empty athena table from the raw file pushed by
     a data producer for raw data.
     """
-    create_glue_database(
-        glue_client, "data_products_raw", logger, bucket, s3_security_opts
-    )
+    database_name = metadata_glue["DatabaseName"]
+    create_glue_database(glue_client, database_name, logger, bucket, s3_security_opts)
 
     # Create raw data table, recreating it if necessary
     table_name = metadata_glue["TableInput"]["Name"]
     try:
-        glue_client.delete_table(DatabaseName="data_products_raw", Name=table_name)
+        glue_client.delete_table(DatabaseName=database_name, Name=table_name)
```

Comment:
This change is broken because when inferring the schema, we use the curated database name, but the raw table name